.github/workflows/build.yaml

```diff
@@ -28,6 +28,12 @@ jobs:
           cache: yarn
       - name: Install
         run: yarn --frozen-lockfile --network-timeout 100000 || yarn --frozen-lockfile --network-timeout 100000 || yarn --frozen-lockfile --network-timeout 100000
+      - name: Create fake contributors
+        uses: 1arp/create-a-file-action@0.2
+        with:
+          path: 'static'
+          file: 'contributors.json'
+          content: "[]"
```

Comment:
Just adding a note here to investigate why `tests/globalSetup.js` doesn't work for CI for creating this file. 👍